prime-router/src/main/kotlin/cli/FileUtilities.kt

```diff
@@ -1,150 +1,150 @@
 package gov.cdc.prime.router.cli
 
-import gov.cdc.prime.router.CovidSender
 import gov.cdc.prime.router.FakeReport
 import gov.cdc.prime.router.FileSource
 import gov.cdc.prime.router.Metadata
 import gov.cdc.prime.router.Report
 import gov.cdc.prime.router.SettingsProvider
+import gov.cdc.prime.router.TopicSender
 import gov.cdc.prime.router.serializers.CsvSerializer
 import gov.cdc.prime.router.serializers.Hl7Serializer
 import java.io.File
 import java.util.Locale
 
-class FileUtilities {
-    companion object {
+/**
```

Comment:
Kotlin supports singletons by creating them as an object instead of creating a class and putting everything into the `companion object`.